src/cli/cli_tcat.cpp

```diff
@@ -121,30 +137,26 @@ static void HandleBleSecureReceive(otInstance               *aInstance,
     IgnoreReturnValue(otBleSecureFlush(aInstance));
 }
 
-template <> otError Tcat::Process<Cmd("vendorid")>(Arg aArgs[])
+template <> otError Tcat::Process<Cmd("advid")>(Arg aArgs[])
```

Comment:
Would you be able to add CLI tags for these commands?  That way they import into the openthread.io Command reference (https://openthread.io/reference/cli/commands)

Examples of how they are done in the other cli files, like here: https://github.com/openthread/openthread/blob/main/src/cli/cli_tcp.cpp#L89

I don't want to hold up the PR on this though, so we can add in a follow up if needed.

Reply:
@Vyrastas Can we create a issue for fixing the documentation? and do it in another PR? 
I can see that doc wise there is https://github.com/openthread/openthread/blob/main/include/openthread/ble_secure.h#L127 another bug that could be handled there.

Reply:
Sure, I've filed an internal bug with my team (at Google) to add tags for them.